integrationtests/self/hotswap_test.go

```diff
@@ -49,7 +49,7 @@ func (ln *fakeClosingListener) Accept(ctx context.Context) (quic.EarlyConnection
 func (ln *fakeClosingListener) Close() error {
 	if atomic.CompareAndSwapInt32(&ln.closed, 0, 1) {
```

Comment:
You can also change `ln.closed` to `atomic.Int32`

Reply:
Good catch! This could be an `atomic.Bool` actually.